src/multihash.rs

```diff
@@ -35,7 +35,27 @@ pub trait MultihashDigest<const S: usize>:
     /// let hash = Code::Sha3_256.digest(b"Hello world!");
     /// println!("{:02x?}", hash);
     /// ```
-    fn digest(&self, input: &[u8]) -> Multihash<S>;
+    fn digest(&self, input: &[u8]) -> Multihash<S> {
+        let mut input = input;
+        self.digest_reader(&mut input).unwrap()
```

Comment:
Same here, please add a comment or `expect()` to make clear why it is OK to use `unwrap()` here.